lametro/views.py

```diff
@@ -282,7 +282,10 @@ def get_context_data(self, **kwargs):
         end_date_str   = self.request.GET.get('to')
         day_grouper    = lambda x: (x.local_start_time.year, x.local_start_time.month, x.local_start_time.day)
 
-        minutes_queryset = EventDocument.objects.filter(note__icontains='minutes')
+        # We only want to display approved minutes
+        minutes_queryset = EventDocument.objects.select_related('event')\
+                                        .filter(note__icontains='minutes')\
+                                        .filter(event__extras__approved_minutes=True)
```

Comment:
It behaves the same way, IIRC, but I prefer combining filter conditions into a single filter call. Can you make that change?

Reply:
No problem.